etc/config-template.toml

```diff
@@ -105,7 +105,9 @@
 # advertise-addr = ""
 
 ## Status address.
-## This is used for reporting the status of TiKV directly through the HTTP address.
+## This is used for reporting the status of TiKV directly through 
+## the HTTP address. Notice that there is a risk of leaking status
+## information if this port is exposed to the public.
```

Comment:
Why not support https directly?

Reply:
I think TLS is another problem, it is handled by another issue #7014 .

The problem is even the TLS is supported, it needs the user to configure the client verification which means signing certificates for clients.

So TLS has two propose here:

1. Encrypt the connection which prevents the middle-man attack.
2. Verify the client which can also be solved by some simpler authentication method(like basic auth).

1  can not prevent information leaks if the port is public, 2 is hard to get enough attention to push users to verify the clients.

Reply:
So to make it secure totally, we can support TLS + basic auth, right?

Reply:
In a k8s deployment with tidb-operator we can automate the setup of certs or basic auth. I don't think we do basic auth elsewhere, so I don't think we should start doing it here without additional thought as to whether this is an approach we would use more uniformly.

Reply:
We can have basic authorization based on CommonName field as suggested in https://github.com/tikv/tikv/issues/6982. tikv-server can be configured to only allow specific client that has the right CN.

Reply:
Yes, this is the TLS auth approach. The CN for this status port would be different than the CN for normal data access, so it needs to be configured separately.